Source/Core/Core/CoreParameter.h

```diff
@@ -257,6 +257,5 @@ struct SCoreStartupParameter
 	static IniFile LoadLocalGameIni(const std::string& id, int revision);
 	static IniFile LoadGameIni(const std::string& id, int revision);
 
-private:
-	static void LoadGameIni(IniFile* game_ini, const std::string& path, const std::string& id, int revision);
+	static std::vector<std::string> SCoreStartupParameter::GetGameIniFilenames(const std::string& id, int revision);
```

Comment:
Buildbot on non-win32 doesn't like this one; you might want to remove the class qualifier